core/oracles/oracle_spec.go

```diff
@@ -72,34 +72,18 @@ type condition func(string) (bool, error)
 // suits the processing of the filters.
 // OracleSpec allows the existence of one and only one.
 func NewOracleSpec(originalSpec types.ExternalDataSourceSpec) (*OracleSpec, error) {
-	signersFromSpec := []*types.Signer{}
-	if originalSpec.Spec != nil {
```

Comment:
I'm not sure you can remove this check - because if originalSpec.Spec or originalSpec.Spec.Data being nil is valid, then it will have a NPE on line 75 in the new code. This is probably only true if someone sends the transaction not using the wallet, but still.

Reply:
This has been addressed earlier, thanks !